docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+Rebuilding an Admin Workstation USB
+-----------------------------------
+
+In cases where an Admin Workstation stick has been lost or destroyed, and no
+backup exists, it is possible to rebuild one. In order to do so, you'll need
+ 
+ - physical access to the SecureDrop servers
+ - 2 USB sticks:
+
+   - Tails Master USB
+   - 1 replacement Admin Workstation USB (USB3 and 16GB or better recommended)
+
+The process requires experience with the Linux command line and Tails, and 
+can take up to 3 hours. If a backup of the SecureDrop application server is available,
+reinstalling the instance may be simpler. An outline of the steps involved is as 
+follows:
+
+ #. Prepare the USB sticks.
+ #. (Optional) Boot the *Application* and *Monitor Server* in single user mode and  reset
+    the shell admin account password.
+ #. Set up SSH access for the new *Admin Workstation*.
+ #. Retrieve SecureDrop configuration settings from the  *Application* and *Monitor Server*.
+ #. Back up and configure the SecureDrop application.
+ #. Run the ``install`` and ``tailsconfig`` commands from the new *Admin Workstation*.
+ #. Complete post-rebuild tasks.
+
+
+Step 1: prepare USB sticks
+==========================
+
+First, create a Tails Master USB using the installation procedure described here: 
+https://tails.boum.org/install/index.en.html
+
+Next, use the Tails Master USB to :ref:`create a new Admin Workstation USB <set_up_tails>` and set up a persistent volume with a strong passphrase.
+
+Once persistence has been set up, start up the Admin Workstation with 
+persistence enabled, :ref:`install the SecureDrop application code, and set up
+the KeePassX database <set_up_admin_tails>`.
+
+The Admin Workstation uses SSH with key authentication to connect to the servers,
+so you'l need to create a new SSH keypair for your SecureDrop instance. To do so,
+open a Terminal and run the following command:
+
+.. code:: sh
+
+ ssh-keygen -t rsa -b 4096
+
+When prompted to ``Enter file in which to save the key``, Press **Enter** to use
+the default location.
+
+
+Step 2: (Optional) Boot the servers in single-user mode
+=======================================================
+If you do not have the original password for the shell admin account on the
+*Apllication* and *Monitor Servers*, you'll need to reset the password on each
+server by booting in single user mode. In order to do so, you'll need physical
+access to the server, a keyboard, and a monitor. 
+
+First, connect a monitor and keyboard to the *Monitor Server*. Then reboot the server.
+When the GRUB menu appears, make sure the **Ubuntu** entry is highlighted, and 
+press **e** to edit boot options.
+In the boot options for Ubuntu, find the line that starts with ``linux`` and ends
+with ``ro``. Add `` single`` after ``ro`` (make sure to include the space!) and press
```

Comment:
Sadly that `single` formatting won't work, leading or trailing whitespace is not permitted in restructured text inline markup. See [ye olde stackoverflowe](https://stackoverflow.com/questions/39151571/how-to-add-leading-trailing-spaces-to-monospace-text-in-sphinx-restructuredtext/39152201) for some possible workarounds, none of which look great :/

Reply:
Note also that we do have instructions with screenshots for booting into single user mode, here:
https://docs.securedrop.org/en/release-0.13.0/kernel_troubleshooting.html#boot-into-single-user-mode

Linking into that section could be confusing, but might be worth turning those into a re-usable content block?

Reply:
It'd need to be edited to remove reference to the new kernel. Edited to fix the code and added a clause to address the space.